terraform/bitbar-devicepool/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+tag_project_name = "bitbar-devicepool"
+
+ssh_user = "relops"
+
+// aerickson's key
+ssh_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCVIzgS4A6o6HZ5QoQvYbRz2Ztwdu1saKSFHz2PfvDiLrhbNzrxekJ+KCYIo0KjmxQGbPFfWYmLQCm1ARdqtU2hBTcm/NQrKZDendLwiX1EGBHvrwy4bwdlm72Hp+O1czb8tyvwgJMfVjmwWMG0FTRHunrv8eFyp9AmSlnDP+BQkEii41f7UY8tdZGbYChLKjHz2px2lhKj1jRt879Vy4zLw9l6wjfJ40A88Cf1rfsblfKP7iS5umeGvtqtSDbOU1dbiH73IJci/GTQiptr/Scu7MBMq/clzD/WGC+cIdregaG7AhZLcr4kqpk2lrAWpfRK/LgHa7vYy8XvRHt+oTpF aerickson@aerickson-45094.local"
```

Comment:
I'd like to see an ssh key variable, maybe a map of all relops keys, located in the common directory so that all projects can reference it.  That would give us a single location to add/remove/update keys with regards to terraform managed infra instead of having to grep for keys and ultimately missing some.

Reply:
Yeah, that would be nice.

We could create a keypair just for bootstrapping and store the private key in the team 1Password. This user isn't supposed to be around for long - maybe it should be sort of annoying to get into (you have to point at a different keyfile). Ideally, puppet would be the one place where we have public keys.

Reply:
OK, let's stick with just my key for now. Putting all relops member's keys in here would be a fair amount of duplication and not easy to do. The shared SSH key has it's own issues.